python/test/t_NLopt_std.py

```diff
@@ -49,6 +54,6 @@ def printNumericalPoint(point, digits):
                     print('algo=', algo)
                     algo.run()
                     result = algo.getResult()
-                    print('x^=', printNumericalPoint(result.getOptimalPoint(), 4))
+                    #print('x^=', printNumericalPoint(result.getOptimalPoint(), 3))
```

Comment:
Because for some NLopt algorithms (the random ones) the output change slightly from run to run. I don't know how to manage NLopt algorithms: we provide access to all the algorithms proposed by the nlopt library, but we don't test them, and if we do we find that some algorithms generate segmentation violation, others provide varying results that can be very far away from the correct result in some cases... Any idea is welcomed!